mobile/src/main/java/rs/readahead/washington/mobile/views/settings/SecuritySettings.kt

```diff
@@ -248,17 +261,63 @@ class SecuritySettings : BaseFragment() {
     }
 
     private fun setUpLockTimeoutText() {
-        binding.lockTimeoutSetting.setText(lockTimeoutManager.selectedStringRes)
+        binding.lockSettingsButton.setLabelText(getString(lockTimeoutManager.selectedStringRes))
     }
 
     private fun onFailedAttemptChoosen(option: Long) {
         failedUnlockManager.setFailedUnlockOption(option)
         TellaKeysUI.setNumFailedAttempts(option)
-        setUpDeleteAfterUnlockText()
+        TellaKeysUI.setRemainingAttempts(failedUnlockManager.getUnlockRemainingAttempts())
+        setUpSettingsVisibility()
     }
 
-    private fun setUpDeleteAfterUnlockText() {
-        binding.deleteUnlockSetting.setText(failedUnlockManager.getFailedUnlockOption())
+    private fun setUpSettingsVisibility() {
+        if (!CamouflageManager.getInstance().isDefaultLauncherActivityAlias) {
```

Comment:
I think it is better to use variable here with Meaningful names                                                                                             
   val isDefaultLauncher = CamouflageManager.getInstance().isDefaultLauncherActivityAlias
    val failedUnlockOption = failedUnlockManager.getFailedUnlockOption()
    val failedUnlockOptionNonZero = failedUnlockOption != 0L
    val launcherName = cm.getLauncherName(baseActivity)